dispatch/static/manager/src/js/api/dispatch.js

```diff
@@ -155,6 +155,14 @@ var DispatchAPI = {
     createTag: (token, name) => {
       return postRequest('tags', null, {name: name}, token)
     }
+  },
+  dashboard: {
```

Comment:
Honestly didn't give it much thought when I built it, figured it'd be easy to move around once the functionality was there. 

/{page}/{data} is probably the most common pattern and i'm not opposed to it. 

Out of habit, I make my endpoints cause this is how i designed my APIs at work cause its a good way to dig through large nested datasets. 
/{resource}/{id/some_other_dimension}

I think for your suggestion is probably better here so i'll change it to that. 👍